website/docs/developer-docs/docs/style-guide.mdx

```diff
@@ -234,25 +251,35 @@ and_backend_roles: []
 description: "Maps admin to all_access"
 ```
 
-- `` ` ` ` yaml `` defines the language used for syntax highlighting. Other languages can be used such as; `jsx`, `python`, `bash` and `text`.
+- ` ` ` `yaml `defines the language used for syntax highlighting. Other languages can be used such as`jsx`, `python`, `bash`, and `text`.
 
 Optional configurations:
 
-- `showLineNumbers`: enables the line numbering on the left-hand side of the code block.
-- `title=" "`: defines the title displayed at the top of the code block. This is useful for defining filenames and paths.
-- `{5}`: defines the line number that will be highlighted. Ranges and lists can also be used (e.g. `{5, 7, 9-11}`).
-- `// highlight-next-line`: is used within a code block to highlight the next line.
-- `// highlight-start` and `// highlight-end`: are used within a code block to highlight one or more lines.
+- `showLineNumbers`: enables line numbering.
+- `title=" "`: defines the title displayed at the top (e.g., filenames).
+- `{5}`: highlights specific lines. Ranges and lists are allowed (e.g., `{5, 7, 9-11}`).
+- `// highlight-next-line`: highlights the next line within a code block.
+- `// highlight-start` and `// highlight-end`: highlight multiple lines.
 
-More information on codeblocks can be found in the [Docusaurus code block documentation](https://docusaurus.io/docs/markdown-features/code-blocks).
+For more details, see the [Docusaurus code block documentation](https://docusaurus.io/docs/markdown-features/code-blocks).
 
-## Component-based formatting
+### Tables
+
+Use tables to compare options, list parameters, or summarize information. Ensure tables are concise and avoid nesting complex content. Only use a table when there are 4 or more items. For only 2 or 3 items, use a bullet list.
 
-This section covers the usage of React components within our documentation. Files that use component-based formatting must use the `.mdx` extension.
+### Lists
```

Comment:
I was wrong, there is a bit of conflict between this PR and the one I worked on today.. I too added a new section to cover Lists... but about the punctuation and capitalization.

Your content is important too; we need a way to add both... let's think on how without having two sections called Lists.